net.clanwolf.starmap.client/src/main/java/net/clanwolf/starmap/client/gui/panes/settings/SettingsPaneController.java

```diff
@@ -517,6 +517,11 @@ public void initialize(URL url, ResourceBundle rb) {
 		checkbox_Sound.setText(Internationalization.getString("app_pane_settings_EnableSound"));
 		checkbox_Music.setText(Internationalization.getString("app_pane_settings_EnableMusic"));
 
+		//Tab 4 Generals
+		tab4.setText(Internationalization.getString("app_pane_settings_tab4Name_Generals"));
+		checkbox_HistoryScreenshot.setText(Internationalization.getString("app_pane_settings_generals_access_clipborad"));
+		checkbox_CheckClipBoardForMWOAPI.setText(Internationalization.getString("app_pane_settings_generals_historyscreenshots"));
```

Comment:
app_pane_settings_generals_access_clipborad und app_pane_settings_generals_access_clipborad müssen jeweils getauscht werden.
Sonst wird es in den Einstellungen falsche angezeigt.